test/fixtures/accounts.js

```diff
@@ -119,7 +119,7 @@ const Account = stampit({
 		this.publicKey =
 			publicKey ||
 			randomstring
-				.generate({ charset: '0123456789ABCDE', length: 32 })
+				.generate({ charset: '0123456789ABCDE', length: 64 })
```

Comment:
And to be on safe side, we are including `lisk-elements` in tests. If we can use it to generate the valid public key for lisk-protocol that will be a safest way to do it.

Reply:
I reckon this is out of the scope of this PR.